frontend/js/integrations/search.js

```diff
@@ -120,9 +127,11 @@ define([
             $.when(user, isAdmin).then(function (userResult, isAdmin) {
                 var user = userResult[0];
                 var userData = user.user;
+                var nickname = (typeof(userData.name) !== "undefined" && userData.name !== null) ? userData.name : userData.username;
+
                 this.user = new User({
                     user_extid: userData.username,
-                    nickname: userData.name || userData.username,
```

Comment:
In our internal questionnaire branch, getting the real nickname was introduced earlier than in the master; this could be done either way i guess, the merge seems to remove masters approach which is a bit cleaner and could persist